app/forms/SelectedSchemesForm.scala

```diff
@@ -46,46 +46,39 @@ object SelectedSchemesForm {
     Scheme("HousesOfParliament", Degree_22, specificRequirement = false),
     Scheme("HumanResources", Degree_22, specificRequirement = false),
     Scheme("ProjectDelivery", Degree_22, specificRequirement = false),
-    Scheme("ScienceAndEngineering", Degree_CharteredEngineer, specificRequirement = true),
-    Scheme("Tax", Degree_22, specificRequirement = false)
```

Comment:
Could you please remove the "Tax" scheme from SchemeType enum in fset-faststream as well.